architecture/pricing-sources-matrix.md

```diff
@@ -60,6 +60,8 @@ kubecostProductConfigs:
     internetNetworkEgress: 0.12 # per GB per month cost
 ```
 
+Setting `customPricesEnabled` to `true` or enabling custom pricing through the UI has been shown to fix an error where Kubecost reported absurdly high cost metrics.
```

Comment:
I don't think this is right; I think this should just be a line about rancher kubernetes engine clusters needing to have custom prices enabled.

Reply:
If this only affects Rancher setups, we should just move this content into this article: https://docs.kubecost.com/install-and-configure/install/provider-installations/istio-rancher

Reply:
afaik it's only rancher.